Library/Formula/openjpeg.rb

```diff
@@ -22,7 +22,13 @@ class Openjpeg < Formula
   depends_on "libpng"
 
   def install
-    system "cmake", ".", *std_cmake_args
+    args = std_cmake_args
+    system "cmake", ".", *args
+    system "make", "install"
```

Comment:
I think that would be difficult for the openjpeg developers to implement in a way that works on all platforms. They would probably suggest the current solution of building the static and shared library separately as most distributions currently do.